R/constants.R

```diff
@@ -179,6 +179,13 @@ aglu.WOOD_ENERGY_CONTENT_GJKG <- 0.0189
 # wood water content
 aglu.WOOD_WATER_CONTENT <- 0.065
 
+# Min veg and soil carbon densities
```

Comment:
@pkyle any memory of aglu.MIN_VEG_CARBON_DENSITY and aglu.MIN_SOIL_CARBON_DENSITY?

I'm guessing (some unit of Carbon mass)/(some land area unit or some mass unit or some volume unit)